### DISCUSSION TITLE
Reordering list of authors

### DISCUSSION THREAD

**Discussion by lmorgan6** [dandi/helpdesk#35] [Ideas]
It would be a nice if we could drag and change the order in which authors were listed. Right now, authors are listed in the order in which the data was input. This makes it hard to order the names in a more informative way after all the authors have been added i.e. put PIs first or last to demonstrate contribution to the dataset. It's helpful for when we want to "cite" the dataset.

**Comment by waxlamp** [comment]
Hi @Imorgan6. It is actually possible to re-order the authors. To do so, click on the "metadata" button to enter the metadata editor, then click on "contributors", which will bring up a panel to edit the author information. It is not super intuitive, but you can actually click-and-drag a full author panel up and down to change the author order.

I think that this should be more discoverable and easier to actually use, but in the meantime you can do it this way to have control over the author list.

Hope this helps!

**Comment by satra** [comment]
i believe @lmorgan6 use case has close to 40 authors, which makes the UI impractical for reordering. In addition, we are working with the author of the component to make this easier. 

for this use case at present i would suggest using the command line tool to reorder authors. happy to provide instructions to do so.
